site/content/entry/yuzu-progress-report-aug-2023/index.md

```diff
@@ -0,0 +1,531 @@
++++
+date = "2023-09-08T12:00:00-03:00"
+title = "Progress Report August 2023"
+author = "GoldenX86"
+coauthor = "CaptV0rt3x"
+forum = 0
++++
+
+[Tuturu~](https://www.youtube.com/watch?v=HkGNeN0LGOE) yuz-ers. Did you feel that the last report was too short? Well, August is back, and with a vengeance! This month offers important graphical fixes, developers working around GPU limitations, several long overdue file system fixes, Skyline framework support, some Android and Apple love, and more!
+
+<!--more--> 
+
+## yuzu Fried Chicken Forever
+
+And Robin.
+
+This is {{< gh-hovercard "11225" "the final piece" >}} of `Project Y.F.C.`, the `Query Cache Rewrite`, and the implementation of `Host Conditional Rendering`. But before we start, we must explain a few things, starting with:
+
+### What is a Query Cache?
+
+Many GPUs have a feature called counters, which are used to gather data from the various commands that GPU executes — things like how many pixels were drawn or how many triangles were generated, etc. 
+Games use "queries" to fetch and load this data into memory whenever they need to use it.
+
+A Query Cache is a cache that emulates the GPU counters through the use of Host GPU queries. 
+yuzu's Query Cache runs these queries on the Host's (user) GPU to emulate the Switch's GPU counters.
+
+### What is it used for?
+
+Some games can make use of this fetched data to further optimise their rendering. For example, `SUPER MARIO ODYSSEY` optimises the amount of triangles it renders using a commonly used technique called "Occlusion Culling".
+
+Occlusion Culling is a feature that disables rendering of objects when they are not currently seen by the camera because they are obscured (occluded) by other objects.
```

Comment:
```suggestion
Occlusion culling is a feature that disables rendering of objects when they are not currently seen by the camera because they are offscreen or obscured (occluded) by other objects.
```